src/main/java/com/oramon/kata/GameOfLife.java

```diff
@@ -0,0 +1,29 @@
+package com.oramon.kata;
+
+import com.oramon.kata.draw.DrawStrategy;
+import com.oramon.kata.draw.impl.DrawStrategyImpl;
+
+/**
+ * ClassName
+ * <p>
+ * Created by Alfredo Gil Rubio
+ * <p>
+ * Date: 26/05/17 11:57
+ */
+public class GameOfLife {
+    private int[][] world;
+    DrawWorldStrategy drawWorldStrategy;
+
+    public GameOfLife(int[][] world,DrawStrategy strategy) {
+        this.world=world;
+        this.drawWorldStrategy=new DrawWorldStrategy(strategy);
+    }
+
+    public int[][] getWorld() {
+        return world;
+    }
+
+    public String printWorld() {
```

Comment:
You have put the logic to print the world in console inside of Game Of Life. Do you think that this class should have this responsability?